### PR TITLE
Historical BEIS data added

### PR DESCRIPTION
**Pull request by deliceli**
**Is this a new addition or an update to an existing factor?**

Update

**What category or categories does it affect?**
- fuel_type_natural_gas-fuel_use_na
- fuel_type_natural_gas-fuel_use_na
- electricity-energy_source_grid_mix_losses
- electricity-energy_source_grid_mix
- water-supply
- water-treatment

**What?**

2016-2020 BEIS data added

**Why?**

Customer request.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -4896,6 +4901,11 @@ ffb8617d-db84-4a65-92da-da7f0706f59e,Energy,Electricity,electricity-energy_sourc
...
+4f6e80ec-d2a4-4d31-884a-8197b19056b0,Energy,Electricity,electricity-energy_source_grid_mix_losses,Transmission & distribution - UK electricity,kWh,not-supplied,0.20052,0.01987,2.4e-06,4.02685e-07,not-supplied,not-supplied,,3,transmission_and_distribution,BEIS,,2020,2020,,GB,,Climatiq,2022/11/10,Emission intensity of transmission & distribution of UK electricity as reported. These factors include only direct emissions at UK power stations and from autogenerators plus those from the proportion of imported electricity. They do not include emissions resulting from production and delivery of fuel to these power stations. Retrieved from the Conversion Factors 2020: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2020
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:4904]: Are these numbers correct? year 2020 looks like an outlier here

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -6839,6 +6849,11 @@ ec9680b6-acb9-44e2-b725-eea68207ebb2,Energy,Fuel,fuel_type_naphtha_net-fuel_use_
...
+6da61464-5789-446a-89b3-d169cdf15a26,Energy,Fuel,fuel_type_natural_gas-fuel_use_na,Natural gas,m3,not-supplied,0.28307,0.28088,2.53846e-05,5.13423e-06,not-supplied,not-supplied,,1,fuel_combustion,BEIS,,2018,2018,,GB,,Climatiq,2022/11/10,Emission intensity of stationary combustion of fuel. Fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Retrieved from the Conversion Factors 2018: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2018
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:6854]: Is this correct? looks like an outlier if I look at other entries

**Review by AnastasiaLobanova — DISMISSED** [review]
Nice job! Just couple of entries look suspicious to me :)

**Review by maryarbab — APPROVED** [review]

**Comment by maryarbab** [comment]
@deliceli Looks clean! Just a tiny note: in the PR description above, please add categories instead of activity IDs.

**Pull request merged**